Decisions.Microsoft365.Exchange/Steps/CalendarSteps.cs

```diff
@@ -17,54 +17,54 @@ public class CalendarSteps
         };
         
         public Microsoft365Event? CreateCalendarEvent(Microsoft365CalendarEvent microsoft365CalendarEvent, string userIdentifier, string? calendarId,
-            [PropertyClassification(0, "Settings Override", "Settings")] ExchangeSettings? settingsOverride)
+            [PropertyClassification(0, "Settings Override", "Settings")] InputExchangeSettings? settingsOverride)
```

Comment:
I would test what happens for an upgrade here. We may actually have to make new methods for all of these steps to be able to update the input without breaking people already using the steps